src/plutolib.c

```diff
@@ -139,14 +140,19 @@ static isl_stat extract_basic_set(__isl_take isl_basic_set *bset, void *user) {
   return isl_stat_ok;
 }
 
+struct pluto_stmt_context_info {
+  Stmt **stmts;
+  PlutoContext *context;
+};
+
 /* Used by libpluto interface. Expects that stmts[idx] have been initialized to
  * NULL.  */
```

Comment:
Doc style /// comment.

Reply:
This is really not documentation but a comment on the working of a local function. So just "// ".